geo-location.html

```diff
@@ -108,6 +114,15 @@
         notify: true,
         readOnly: true,
         value: null
+      },
+
+      /**
+       * [Status](https://w3c.github.io/permissions/#status-of-a-permission) of Geolocation API permission.
+       */
+      permissionStatus: {
+        type: String,
+        readOnly: true,
+        notify: true
```

Comment:
needs a default state.

Reply:
@ebidel Is not the default value optional? Should I use `value: null`?

Reply:
`null` sounds good. As long as that's not one of the possible values the API returns.

Reply:
I mean that if we do not specify a default value at all, it is equivalent to `null`. Should I specify it explicitly?